### ISSUE TITLE
Outdated WebSocket options documentation 

### ISSUE THREAD

**Issue by sergio-bershadsky** [crossbario/crossbar#1830]
https://crossbar.io/docs/WebSocket-Options/

It says that all timeouts are specified in `ms` that are 1/1000 of a second but actual timeouts are in seconds.

**Comment by sergio-bershadsky** [comment]
See: utobahn/asyncio/wamp.py:214

**Comment by sergio-bershadsky** [comment]
Remark. If this options are specified via json file for crossbar router - values in ms. But if options passed via Component __init__ values are seconds

**Comment by oberstet** [comment]
yes, that's unfortunately true: https://autobahn.readthedocs.io/en/latest/websocket/programming.html?highlight=timeout#common-options-server-and-client
